osquery/tables/system/windows/userassist.h

```diff
@@ -17,6 +17,6 @@ namespace tables {
 std::string rotDecode(std::string& value_key_reg);
 
 // Get Epoch time from Windows FILETIME
-auto lastExecute(std::string& time_data);
+long long lastExecute(std::string& time_data);
```

Comment:
Heads up that this API is 'rare', it is not clear to the calling function that `time_data` will be modified. I recommend add more documentation or changing the parameter to be a const reference then mutating a copy within the function.

I also think we should move these utility functions into a utility library but that can be handled as a follow up.

Reply:
So I made few changes to the function:

- Renamed to `littleEndianToUnixTime`. It can be used to convert a little endian Windows FILETIME string to unix/epoch time
- Added `const` and mutate a copy in the function

I can open another PR to move this into a utility, and i can probably add a test for it to